cleanlab/internal/regression_utils.py

```diff
@@ -0,0 +1,28 @@
+"""
+Helper function internally used in cleanlab.regression
+"""
+
+import numpy as np
+
+
+def assert_valid_inputs(
+    labels: np.ndarray,
+    predictions: np.ndarray,
```

Comment:
@elisno why would there be `pred_probs` for regression?

IMO the confusion that is more likely to happen here is that somebody just sees `assert_valid_inputs()` in some code and doesn't realize it is for regression rather than classification. To mitigate, could consider renaming this function: `assert_valid_regression_inputs()`  but I don't have strong opinions either way